tests/spam_test.py

```diff
@@ -7,8 +7,17 @@
 class TestSpam(unittest.TestCase):
 
     def setUp(self):
+        extra_config = {
+            'DEFAULT_CONFIG': {
+                'SpammingAlert': {
+                    'MAX_MSG_LEN': 1000,
+                    'MAX_LINES': 10,
+                }
+            }
+        }
         self.testbot = TestBot(extra_plugin_dir='plugins',
-                               loglevel=logging.ERROR)
+                               loglevel=logging.ERROR,
+                               extra_config=extra_config)
```

Comment:
extra config here should override the plugin defaults, so these tests passing show that your code is not working correctly.

Anyway, dont add `extra_config` here.  This test class is good without it.

Create an additional test class `TestSpamExtraConfig` which verifies the extra config overrides the plugin defaults.

Reply:
done